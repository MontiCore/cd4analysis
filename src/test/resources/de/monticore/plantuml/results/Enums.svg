<?xml version="1.0" encoding="us-ascii" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentStyleType="text/css" height="260px" preserveAspectRatio="none" style="width:208px;height:260px;background:#FFFFFF;" version="1.1" viewBox="0 0 208 260" width="208px" zoomAndPan="magnify"><defs/><g><rect fill="none" height="27.6094" id="_legend" style="stroke:#000000;stroke-width:1.0;" width="81" x="93.5" y="12"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="71" x="98.5" y="31.5332">.Enums CD</text><!--class State--><g id="elem_State"><rect codeLine="19" fill="#FFFFFF" height="88.4375" id="State" style="stroke:#181818;stroke-width:0.5;" width="68" x="20.5" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="33" x="38" y="79.1426">State</text><line style="stroke:#181818;stroke-width:0.5;" x1="21.5" x2="87.5" y1="87.2188" y2="87.2188"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="38" x="26.5" y="105.752">DEAD</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="38" x="26.5" y="123.3613">ALIVE</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="56" x="26.5" y="140.9707">UNDEAD</text></g><!--class Color--><g id="elem_Color"><rect codeLine="25" fill="#FFFFFF" height="88.4375" id="Color" style="stroke:#181818;stroke-width:0.5;" width="70" x="123.5" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="34" x="141.5" y="79.1426">Color</text><line style="stroke:#181818;stroke-width:0.5;" x1="124.5" x2="192.5" y1="87.2188" y2="87.2188"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="29" x="129.5" y="105.752">RED</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="35" x="129.5" y="123.3613">BLUE</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="58" x="129.5" y="140.9707">YELLOW</text></g><!--class Person--><g id="elem_Person"><rect codeLine="31" fill="#FFFFFF" height="27.6094" id="Person" style="stroke:#181818;stroke-width:0.5;" width="51" x="29" y="207.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="45" x="32" y="227.1426">Person</text></g><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacing" textLength="200" x="0" y="250.5996">generated with MontiCore using PlantUML</text><!--SRC=[RL3DQiCm33x7Jw4dsFO0PP0sEHJIhhHaPKStqH9H_mHRONIZxpuvxcKm7opfqoVfarQHRmPVLT4P7MD5UWZQmeygCh1unLftro7usVKRRto0yqYCAN7CM70Ou4KL0FVq6HpGzL8Ao_k_EkV-DaZ8NQsU7vf6wX4w2ft1xAET-0OMxGL3L4AFLtAJJeeNjILpeClCkEsyEz4tIi-76FOJ11f6bggdsiqsmgPIAPC9uiuMJgmP5-5LNLPYocRxNejjzmjmLvcN5qcyOvrewwPDh8-wQTxEYPMFTn2Ptac4ZcN1J- -JaN4OP4uFNyGZxBnZaXCWp97S00UZ7RUxvXS0]--></g></svg>
