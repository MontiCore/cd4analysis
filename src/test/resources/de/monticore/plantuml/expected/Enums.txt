@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .Enums CD
  end legend
  enum State {
    DEAD
    ALIVE
    UNDEAD
  }

  enum Color {
    RED
    BLUE
    YELLOW
  }

  class Person {
    {field} State state
    {field} Color color
  }

  center footer generated with MontiCore using PlantUML
@enduml
